ionoscloud/resource_autoscaling_group.go

```diff
@@ -0,0 +1,483 @@
+			//	Optional:         true,
+			//	Computed:         true,
+			//	ValidateDiagFunc: validation.ToDiagFunc(validation.IntBetween(0, 200)),
+			//},
+			"name": {
+				Type:             schema.TypeString,
+				Description:      "User-defined name for the Autoscaling Group.",
+				Required:         true,
+				ValidateDiagFunc: validation.ToDiagFunc(validation.StringLenBetween(1, 255)),
+			},
+			"policy": {
+				Type:        schema.TypeList,
+				MaxItems:    1,
+				Description: "Defines the behavior of this VM Auto Scaling Group. A policy consists of triggers and actions, where an action is an automated behavior, and the trigger defines the circumstances under which the action is triggered. Currently, two separate actions are supported, namely scaling inward and outward, triggered by the thresholds defined for a particular metric.",
+				Required:    true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"metric": {
+							Required:         true,
+							Type:             schema.TypeString,
+							Description:      "The Metric that should trigger the scaling actions. Metric values are checked at fixed intervals.",
+							ValidateDiagFunc: validation.ToDiagFunc(validation.StringInSlice([]string{"INSTANCE_CPU_UTILIZATION_AVERAGE", "INSTANCE_NETWORK_IN_BYTES", "INSTANCE_NETWORK_IN_PACKETS", "INSTANCE_NETWORK_OUT_BYTES", "INSTANCE_NETWORK_OUT_PACKETS"}, true)),
+						},
+						"range": {
+							Optional:    true,
+							Computed:    true,
+							Type:        schema.TypeString,
+							Description: "Specifies the time range for which the samples are to be aggregated. Must be >= 2 minutes.",
+						},
+						"scale_in_action": {
+							Required:    true,
+							Type:        schema.TypeList,
+							MaxItems:    1,
+							Description: "Specifies the action to take when the `scaleInThreshold` is exceeded. Hereby, scaling in is always about removing VMs that are currently associated with this autoscaling group. Default termination policy is OLDEST_SERVER_FIRST.",
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"amount": {
+										Required:    true,
+										Type:        schema.TypeInt,
+										Description: "When `amountType == ABSOLUTE`, this is the number of VMs added or removed in one step. When `amountType == PERCENTAGE`, this is a percentage value, which will be applied to the autoscaling group's current `targetReplicaCount` in order to derive the number of VMs that will be added or removed in one step. There will always be at least one VM added or removed. For SCALE_IN operation now volumes are NOT deleted after the server deletion.",
```

Comment:
@cristiGuranIonos same thing here, in the doc, the highlighting works properly but here in the description we should use simple quotes, for `targetReplicaCount` for example.